jest.config.js

```diff
@@ -40,6 +41,7 @@ module.exports = {
   setupFilesAfterEnv: ['<rootDir>/test/jest/setup.js'],
   testMatch: [
     '<rootDir>/app/scripts/constants/error-utils.test.js',
+    '<rootDir>/app/scripts/controllers/app-metadata.test.ts',
     '<rootDir>/app/scripts/controllers/app-state.test.js',
     '<rootDir>/app/scripts/controllers/transactions/etherscan.test.ts',
     '<rootDir>/app/scripts/controllers/transactions/EtherscanRemoteTransactionSource.test.ts',
```

Comment:
[out-of-scope] I think we could remove these `controllers/transactions` references